webpack/assets/javascripts/foreman_tools.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
```

Comment:
`module.exports` is the CommonJS (node) way of doing it, `export` is the ES6 way. For more info on the subtle difference: http://www.2ality.com/2015/07/es6-module-exports.html